ang/civicase/case/actions/directives/case-actions.directive.js

```diff
@@ -310,11 +310,19 @@
           },
 
           gotoWebform: function (selectedCase, path, clientId) {
-            clientId = 'cid' + clientId;
-            var url = CRM.url(path, {
-              case1: selectedCase.id,
-              [clientId]: selectedCase.client[0].contact_id
-            });
+            var url;
+            if (clientId) {
+              clientId = 'cid' + clientId;
+              url = CRM.url(path, {
+                case1: selectedCase.id,
+                [clientId]: selectedCase.client[0].contact_id
```

Comment:
`[clientId]` is an ES6 command, and we are not using ES6 yet. So please use ES5 statements.